### PR TITLE
XW-849 | server/app files renamed to dashed-case

### PR DESCRIPTION
**Pull request by Warkot**
CHANGE ALL THE NAMES

## Links

* http://wikia-inc.atlassian.net/browse/JIRA-849

## Description

Dashed-case filenames are cool. For example they're system agnostic (e.g. for Linux file `FOO.JS` is different than `foo.js`, but for Windows it's the same file).

## Reviewers

@Wikia/x-wing

### CODE REVIEW

File: front/auth/tests/unit/common/cookie.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-QUnit.module('auth/common/Cookie', function (hooks) {
+QUnit.module('auth/app/common/cookie', function (hooks) {
```
Comment (vforge) [inline front/auth/tests/unit/common/cookie.js]: `app/`?

Comment (bognix) [inline front/auth/tests/unit/common/cookie.js]: yes, this path is correct

File: front/main/app/instance-initializers/setup-tracking.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
-import Ads from 'common/modules/Ads';
-import UniversalAnalytics from 'common/modules/Trackers/UniversalAnalytics';
+import Ads from 'common/modules/ads';
+import UniversalAnalytics from 'common/modules/trackers/universal-analytics';
 import {integrateOptimizelyWithUA} from 'common/utils/variantTesting';
```
Comment (rogatty) [inline front/main/app/instance-initializers/setup-tracking.js]: `variant-testing`?

**Comment by hakubo** [comment]
Guideline: https://github.com/Wikia/guidelines/pull/109

**Pull request merged**